tools/preseal/Makefile

```diff
@@ -1,9 +1,21 @@
-CC = gcc
-CFLAGS:=
+-include ../../.config
 
-ifneq ($(NO_FILESYSTEM),)
+CC      = gcc
+WOLFBOOTDIR = ../..
+WOLFDIR = $(WOLFBOOTDIR)/lib/wolfssl/
+WOLFTPMDIR = $(WOLFBOOTDIR)/lib/wolfTPM/
+CFLAGS  = -Wall -Wextra -Werror
+CFLAGS  += -I. -DWOLFSSL_USER_SETTINGS -DWOLFTPM_USER_SETTINGS -I$(WOLFDIR) -I$(WOLFBOOTDIR)/include
```

Comment:
missing `-I$(WOLFTPMDIR)` in CFLAGS

Reply:
... And -I$(WOLFBOOTDIR)/include should be removed, or you are including wolfBoot's `user_settings.h` instead of the local one that seems to be created for the tool